src/middleware/initiate.js

```diff
@@ -30,14 +124,42 @@ exports.initiateContextMiddleware = () => async (ctx, next) => {
     return
   }
 
+  const endpointState = await readStateByEndpointId(endpoint._id)
+
+  logger.info(`${endpoint.name} (${requestUUID}): Initiating new request`)
```

Comment:
Minor, but maybe this could be a logger.debug? No data has been altered yet...

Reply:
I left it in cause i though it might be useful to see that a new request came into the mapper. But maybe this isnt really needed?